string encrypted_data
string data_type  # "x", "y", "sum" 등을 구분하기 위한 필드 